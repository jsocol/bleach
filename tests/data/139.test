<div id="68"><doc xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:html="http://www.w3.org/1999/xhtml">
    <html:style /><x xlink:href="javascript:alert(68)" xlink:type="simple">XXX</x>
</doc>//["'`-->]]>]</div>
