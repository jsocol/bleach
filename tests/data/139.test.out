<div id="68">
    XXX
//["'`--&gt;]]&gt;]</div>
